package.json

```diff
@@ -86,5 +86,12 @@
     "slate-react": "*",
     "source-map-loader": "^0.2.4",
     "typescript": "^3.7.2"
+  },
+  "resolutions": {
+    "bl": "^2.2.1",
+    "dot-prop": "^4.2.1",
+    "kind-of": "^6.0.3",
+    "minimist": "^0.2.1",
+    "serialize-javascript": "^4.0.0"
```

Comment:
There is a webpack 4.44.0, this resolved the `serialise-javascript` 2.x dependency (and possibly others) for us. Specifically v1.4.4 of `terser-webpack-plugin` upgrades to `serialise-javascript` 3.1.0 fixing the security issue.

Reply:
oh webpack is coming in via `next`, sigh. I don't know if their most recent version includes the newer webpack

Reply:
It doesn't I've updated every package possible :) other than prettier and TS

Reply:
Hmm so #3845 should remove the need to override `serialize-javascript` because it upgrades `next` (which does upgrade webpack). Looking through the yarn lock, nothing depends on the old version yet it's still listed.

Yarn isn't perfect, try regenerating the lock file and see if that removes any of the old dependencies (at the very least it will remove `serialize-javascript@2.1.2`).

Reply:
I can try once that PR gets merged. I think there was still a dependency... but its a bit moot juggling multiple PRs until they get merged.